appcues/src/main/java/com/appcues/Appcues.kt

```diff
@@ -35,16 +37,19 @@ public fun Appcues(
     accountId: String,
     applicationId: String,
     config: (AppcuesConfig.() -> Unit)? = null,
-): Appcues = AppcuesKoinContext
-    // This creates the Koin Scope and initializes the Appcues instance within, then returns the Appcues instance
+): Appcues = Bootstrap
+    // This creates the Scope and initializes the Appcues instance within, then returns the Appcues instance
     // ready to go with the necessary dependency configuration in its scope.
-    .createAppcuesScope(context, AppcuesConfig(accountId, applicationId).apply { config?.invoke(this) }).get()
+    .createScope(
+        context = context,
+        config = AppcuesConfig(accountId, applicationId).apply { config?.invoke(this) }
+    ).get()
 
 /**
  * The main entry point for using Appcues functionality in your application - tracking
  * analytics and rendering experiences.
  */
-public class Appcues internal constructor(koinScope: Scope) {
+public class Appcues internal constructor(internal val scope: AppcuesScope) {
```

Comment:
having scope as internal enables us to grab it for [testing](https://github.com/appcues/appcues-android-sdk/pull/500/files#diff-607cac65757cc76bdcac417970017a2c8ae7cacd8ce651e79dda5f190ec49cd5R55).